tests/unit/logs/test_transformation.py

```diff
@@ -22,6 +22,14 @@
 from util.context import Context
 
 
+def get_context(log_forwarder_setup="log.forwarder"):
+    return Context("function-name", "dt-url", "dt-token", False, False, log_forwarder_setup)
+
+
+def get_batch_metadata():
+    return BatchMetadata("444000444", "us-east-1", "aws")
```

Comment:
If it's always the same (non-parameterized function), maybe let's just create a variable here?

Reply:
I agree that it's better to create a variable here, since it will not change.